pkgs/development/python-modules/textual-dev/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, click
+, fetchFromGitHub
+, msgpack
+, poetry-core
+, pythonOlder
+, textual
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "textual-dev";
+  version = "1.2.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Textualize";
+    repo = "textual-dev";
+    rev = "fb5aa869784895066dbd0ec850d92ac9205cde73";
```

Comment:
Looking at [the release date of pypi](https://pypi.org/project/textual-dev/), I think we should adopt this commit.
https://github.com/Textualize/textual-dev/commit/6afa9013a42cb18e9105e49d6a56874097f7c812

Then could you please add a brief comment about why we don't use a tag, and if possible, ask the upstream to add tags?

Reply:
I've added a comment on my latest push and also asked upstream [here](https://github.com/Textualize/textual-dev/issues/26)